### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by JoseEspinosa**
Some fixes:
* Add the ip and control labels to the plotfingerprint.
* Correctly assign `ch_picardcollectmultiplemetrics_multiqc` channel
* Correct pvalue and qvalue arguments for macs2 callpeak
* Fix `skip_peak_annotation` and `skip_peak_qc options`
* Run preseq on the bam coming from markduplicates and not after filtering orphans
* Fix [#282](https://github.com/nf-core/chipseq/issues/282) make `genome.fa` available for IGV
* Skip `preseq` on `test.config`

### CODE REVIEW

File: conf/test.config [hunk 1]
```diff
@@ -23,12 +23,16 @@ params {
     input = 'https://raw.githubusercontent.com/nf-core/test-datasets/chipseq/samplesheet/v2.0/samplesheet_test.csv'
 
     // Genome references
-    fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/reference/genome.fa'
-    gtf   = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/reference/genes.gtf'
+    genome = 'hg19'
+    // fasta = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/reference/genome.fa'
+    // gtf   = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/reference/genes.gtf'
```
Comment (drpatelh) [inline conf/test.config:28]: These are commented out?

File: workflows/chipseq.nf [hunk 2]
```diff
@@ -484,7 +486,7 @@ workflow CHIPSEQ {
                         meta.replicates_exist = groups.max { groups.value }.value > 1
                         [ meta, peaks ] }
                 .set { ch_antibody_peaks }
-
+            // ch_antibody_peaks.dump()
```
Comment (drpatelh) [inline workflows/chipseq.nf:489]:
```suggestion
```

File: workflows/chipseq.nf [hunk 3]
```diff
@@ -508,14 +510,18 @@ workflow CHIPSEQ {
                 .map { meta, saf -> [ meta.id, meta, saf ] }
                 .set { ch_ip_saf }
 
+            // ch_ip_saf.dump()
+
```
Comment (drpatelh) [inline workflows/chipseq.nf:514]:
```suggestion
```

File: workflows/chipseq.nf [hunk 4]
```diff
@@ -508,14 +510,18 @@ workflow CHIPSEQ {
...
             ch_ip_control_bam
                 .map { meta, ip_bam, control_bam -> [ meta.antibody, meta, ip_bam ] }
                 .groupTuple()
                 .map { it -> [ it[0], it[1][0], it[2].flatten().sort() ] }
+                // .dump()
```
Comment (drpatelh) [inline workflows/chipseq.nf:519]:
```suggestion
```

File: workflows/chipseq.nf [hunk 5]
```diff
@@ -508,14 +510,18 @@ workflow CHIPSEQ {
...
                 .join(ch_ip_saf)
+                .dump()
```
Comment (drpatelh) [inline workflows/chipseq.nf:521]:
```suggestion
```

File: conf/test.config (hunk 1 above)
Comment (JoseEspinosa) [reply conf/test.config:28]: Indeed, that is a leak from some test I run on my local machine 😓

**Comment by JoseEspinosa** [comment]
All comments have been addressed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)

File: conf/test.config (hunk 1 above)
Comment (JoseEspinosa) [reply conf/test.config]: Fixed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)

File: workflows/chipseq.nf (hunk 2 above)
Comment (JoseEspinosa) [reply workflows/chipseq.nf]: Fixed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)

File: workflows/chipseq.nf (hunk 4 above)
Comment (JoseEspinosa) [reply workflows/chipseq.nf]: Fixed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)

File: workflows/chipseq.nf (hunk 3 above)
Comment (JoseEspinosa) [reply workflows/chipseq.nf]: Fixed in [277bd11](https://github.com/nf-core/chipseq/pull/279/commits/277bd119f4747960d0059df30f7a3cea8fccb625)

**Review by drpatelh — APPROVED** [review]
🚀

**Pull request merged**